h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoModel.java

```diff
@@ -18,7 +18,7 @@
     private static final int NsdNaLeft = NaSplitDir.NALeft.value();
     private static final int NsdLeft = NaSplitDir.Left.value();
 
-    protected Number _mojo_version;
+    public Number _mojo_version;
```

Comment:
Why is this needed? It shouldn't be, IMHO

Reply:
I need it to check the mojo version.  Leaf node assignment is only available after Pasha change. I don't need to check it at the outside.  Reverting it back now.